packages/e2e/src/helpers/messages.helper.ts

```diff
@@ -1,5 +1,6 @@
 import MessagesPage from "../page-objects/messages.page"
 import MessagesConversationPage from "../page-objects/messages-conversation.page"
+import ModalMessagesPage from "../page-objects/modal-messages.page"
 
 /** Send message using send button */
 export const sendMessage = async (
```

Comment:
More a question than a suggestion. Why did you decide to move sendMessage and deleteConversationOnThreadList functions from MessagePage to helper?  I asking because those actions are available only on Message Page.

Reply:
It is because pages focus on a part of app UI while helper can use selectors from few different pages (for example Delete message involves also confirmation available on separate modal page